yarn.lock

```diff
@@ -2002,6 +2002,7 @@ __metadata:
     "@babel/core": 7.20.12
     "@babel/preset-env": 7.20.2
     "@babel/preset-typescript": 7.18.6
+    "@graphql-codegen/add": ^5.0.3
```

Comment:
Please add it to someone else. It may break the open source in a future. Can you add it to keystone package too?

Reply:
where? I use it in my app, I've added it in my app.